src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -125,19 +133,25 @@ private bool NeedsRebuilding(ProjectContext project, ProjectDependenciesFacade d
 
             if (!newInputs.Any())
             {
-                Reporter.Output.WriteLine($"\nProject {project.ProjectName()} was previously compiled. Skipping compilation.");
+                Reporter.Output.WriteLine($"Project {project.GetDisplayName()} was previously compiled. Skipping compilation.");
```

Comment:
Kinda unrelated, but should the calls above be `GetLastWriteTimeUtc`? Seems alright to use the non-UTC time below for display, but shouldn't need it for calculating this.